public/js/pimcore/element/helpers/gridCellEditor.js

```diff
@@ -91,9 +91,20 @@ Ext.define('pimcore.element.helpers.gridCellEditor', {
             bodyStyle: "padding: 10px;"
         });
         let width = 700;
-        if (tagType === 'manyToManyObjectRelation' && fieldInfo.layout.width && fieldInfo.layout.width !== '100%') {
+
+        if (tagType === 'manyToOneRelation') {
+            width = sumWidths(
+                formPanel.config.items[0].items.items[0].width,
```

Comment:
this is like this because of a "hidden" default, so getting the `href.width` would make it match 1:1
https://github.com/pimcore/admin-ui-classic-bundle/blob/1a6a1aff055f20e475189b98cbe5b180e93bf32c/public/js/pimcore/object/tags/manyToOneRelation.js#L160